Samples/LibVLCSharp.UWP.Sample/MainPage.xaml

```diff
@@ -8,8 +8,11 @@
     xmlns:lvs="using:LibVLCSharp.Platforms.UWP"
     mc:Ignorable="d"
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <Page.DataContext>
+        <local:MainViewModel />
+    </Page.DataContext>
 
     <Grid>
-      <lvs:VideoView x:Name="VideoView"></lvs:VideoView>
+      <lvs:VideoView SwapChainOptions="{Binding SwapChainOptions, Mode=TwoWay}" MediaPlayer="{Binding MediaPlayer}"></lvs:VideoView>
```

Comment:
Why `TwoWay` binding mode?

Reply:
I would like to use `OneWayToSource` binding mode to send the `SwapChainOptions` to the ViewModel, but `OneWayToSource` binding doesn't exist in UWP. So, I used `TwoWay` binding mode.

Perhaps it might be better to create an event with the SwapChainOptions in the event data, and I can keep a non-bindable SwapChainOptions property.

Reply:
Yes, an event will probably be better (I usually try to avoid `TwoWay` binding mode).